src/components/Lesson/Lesson.js

```diff
@@ -44,6 +46,20 @@ const createMarkup = (lessonContent) => {
   return ({__html: removeHtmlFileEnding(processContent(lessonContent, contentStyles))});
 };
 
+const Progress = ({checkedCheckboxes, totalCheckboxes}) => {
+  const now = totalCheckboxes > 0 ? 100 * checkedCheckboxes / totalCheckboxes : 0;
+  const bsStyle = 'success';
+  const className = styles.progressBar;
+  const label = `${checkedCheckboxes}/${totalCheckboxes}`;
```

Comment:
Bare et forslag: Hva med å ha en checkmark ved siden av tallene? F.eks. `✓ 16/31`?
Og skulle vi satt på en stjerne til høyre for progressbar, eller en gullramme rundt, e.l., når man har checked alle?

Igjen, bare forslag...